File: backend/testing/corpus/javascript/archivo1.js
// Calcula operaciones matemáticas básicas sin entrada externa

function sumar(a, b) {
    return a + b;
}

function multiplicar(a, b) {
    return a * b;
}

function mostrarResultados() {
    const resultadoSuma = sumar(10, 5);
    const resultadoMultiplicacion = multiplicar(4, 3);

    console.log("Resultado suma:", resultadoSuma);
    console.log("Resultado multiplicación:", resultadoMultiplicacion);
}

mostrarResultados();

File: backend/testing/corpus/python/archivo1.py

# Operaciones matemáticas básicas sin entrada externa

def sumar(a: int, b: int) -> int:
    return a + b

def dividir(a: int, b: int) -> float:
    return a / b

def mostrar_resultados():
    resultado_suma = sumar(7, 3)
    resultado_division = dividir(10, 2)

    print("Resultado suma:", resultado_suma)
    print("Resultado división:", resultado_division)

if __name__ == "__main__":
    mostrar_resultados()





